docs/open-telemetry.asciidoc

```diff
@@ -326,9 +327,10 @@ Here is an example of AWS Lambda Java function managed with Terraform and the ht
 
 NOTE: For a better startup time, we recommend using SDK-based instrumentation for manual instrumentation of the code rather than auto instrumentation.
 
+// lint ignore js
 To instrument AWS Lambda Node.js functions, see https://aws-otel.github.io/docs/getting-started/lambda/lambda-js[AWS Distro for OpenTelemetry Lambda Support For JS].
```

Comment:
Instead of the lint ignore can we change the alt text to "JavaScript"?

```suggestion
To instrument AWS Lambda Node.js functions, see https://aws-otel.github.io/docs/getting-started/lambda/lambda-js[AWS Distro for OpenTelemetry Lambda Support For JavaScript].
```